cmd/ak/cmd/orgs/create.go

```diff
@@ -0,0 +1,38 @@
+package orgs
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+var displayName string
+
+var createCmd = common.StandardCommand(&cobra.Command{
+	Use:     "create [--display-name display-name]",
```

Comment:
IIUC, this is a required single field - if so, it's better to use an arg

Reply:
it's not a required field. you can create an unnamed org.

Reply:
so an optional arg?

Reply:
we might add more stuff in the future. i think this should be a --thingy.